test/helpers/TestParachain.sol

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Do you need to make an entire new contract that wraps all the internal functions?
Or could you just `vm.prank(parachainContractItself)`, then call the internal function you need?

Reply:
When using `vm.prank(parachainContractItself)`, the test file still doesn't have the ability to call the internal function on another contract. Maybe there's one more step needed to expose internal functions to the test file?